omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -31,25 +35,46 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
                 emr.errorAlert("Error Saving condition");
             });
         }
+    }
 
     self.initCondition = self.initCondition || function () {
             $scope.patientUuid = CommonFunctions.extractUrlArgs(window.location.search)['patientId'];
             $scope.condition = new ConditionModel($scope.patientUuid);
+            self.showEndDate();
         }
 
-    self.validateCondition = self.validateCondition || function () {
```

Comment:
Why was this removed?

Reply:
This function is not being removed. It is being implemented after showEndDate function.